be/src/exprs/table_function/table_function_factory.cpp

```diff
@@ -77,6 +78,12 @@ TableFunctionResolver::TableFunctionResolver() {
     add_function_mapping("unnest", {TYPE_ARRAY}, {TYPE_BOOLEAN}, func_unnest);
     add_function_mapping("unnest", {TYPE_ARRAY}, {TYPE_ARRAY}, func_unnest);
 
+    // Use special invalid as the parameter of multi_unnest, because multi_unnest is a variable parameter function,
+    // and there is no special treatment for different types of input parameters,
+    // this is just for compatibility and find the corresponding function
+    TableFunctionPtr multi_unnest = std::make_shared<MultiUnnest>();
+    add_function_mapping("unnest", {INVALID_TYPE}, {INVALID_TYPE}, multi_unnest);
```

Comment:
why use invalid type? which is very strange.

Reply:
done